controllers/openshift_controller.go

```diff
@@ -473,6 +474,21 @@ func (r *KataConfigOpenShiftReconciler) newMCForCR(machinePool string) (*mcfgv1.
 		return nil, err
 	}
 
+	// RHCOS uses "sandboxed-containers" as thats resolved/translated in the machine-config-operator to "kata-containers"
+	// FCOS however does not get any translation in the machine-config-operator so we need to
+	// send in "kata-containers".
+	// Both are later send to rpm-ostree for installation.
+	//
+	// As RHCOS is rather special variant, use "kata-containers" by default.
+	var extensions = []string{"kata-containers"}
+
+	if r.Os.IsEL() {
+		extensions = []string{"sandboxed-containers"}
+	}
+	if r.Os.IsFCOS() {
```

Comment:
Is this condition needed since default for extensions is "kata-containers" ?

Reply:
Not really, just wanted to be explicit. I'll remove it.

Reply:
You can add a comment mentioning that the default applies to FCOS.

Reply:
I've removed the if-clause and added a comment that it applies to FCOS.

Reply:
@bpradipt Do you think we could resolve this conversation?

Reply:
Yeah. Thanks @smuda